### PR TITLE
generic_panda/panda65_01 keyboard firmware addition

### PR DESCRIPTION
**Pull request by JesseLeung97**
## Description

Added firmware files used on the panda65_01 65% PCB.  Includes both standard QMK and VIA compatible files.  Has been compiled, flashed, and tested for accuracy and functionality on physical PCB prototypes.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/generic_panda/panda65_01/rules.mk [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+LAYOUTS = split_bs
```
Comment (fauxpark) [inline keyboards/generic_panda/panda65_01/rules.mk:21]:
```suggestion
```

File: keyboards/generic_panda/panda65_01/rules.mk [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# MCU name
+MCU = STM32F072
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no       # Mouse keys
```
Comment (fauxpark) [inline keyboards/generic_panda/panda65_01/rules.mk:8]:
```suggestion
MOUSEKEY_ENABLE = no        # Mouse keys
```

File: keyboards/generic_panda/panda65_01/readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    make Panda65_01:default
+
+Make example for this keyboard's VIA keymap (after setting up your build environment):
+    
+    make Panda65_01:via
+
+Flashing example for this keyboard:
+
+    make Panda65_01:default:flash
```
Comment (fauxpark) [inline keyboards/generic_panda/panda65_01/readme.md:20]:
```suggestion
    make generic_panda/panda65_01:default

Flashing example for this keyboard:

    make generic_panda/panda65_01:default:flash
```

File: keyboards/generic_panda/panda65_01/readme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Panda65_01
+
+![Panda_01](https://i.imgur.com/fPBUDMT.png)
+
+
```
Comment (fauxpark) [inline keyboards/generic_panda/panda65_01/readme.md:5]:
```suggestion
```

File: keyboards/generic_panda/panda65_01/keymaps/via/rules.mk [hunk 5]
```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes # enable via
+LTO_ENABLE = yes # reduce firmware size
+EXTRAKEY_ENABLE = no # diable extra key for via
+MIDI_ENABLE = no #diable midi for via
```
Comment (fauxpark) [inline keyboards/generic_panda/panda65_01/keymaps/via/rules.mk:4]:
```suggestion
VIA_ENABLE = yes
LTO_ENABLE = yes
```

File: keyboards/generic_panda/panda65_01/info.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Panda65_01",
+    "url": "https://github.com/JesseLeung97/qmk_firmware/tree/master/keyboards/generic_panda/panda65_01",
+    "maintainer": "generic_panda",
+    "width": 16,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
```
Comment (fauxpark) [inline keyboards/generic_panda/panda65_01/info.json:8]: This file doesn't match the layout macros.

File: keyboards/generic_panda/panda65_01/readme.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+* Hardware Supported: panda65_01 pcb
+* Hardware Availability: [panda65-01 acrylic case designed by Alec:](https://onedrive.live.com/?authkey=%21AA8d6frZ0woiaV4&id=DC069E9FB0BC8004%21155830&cid=DC069E9FB0BC8004) 
+
+Make example for this keyboard's default keymap (after setting up your build environment):
+
+    make Panda65_01:default
+
+Flashing example for this keyboard:
+
+    make Panda65_01:default:flash
```
Comment (fauxpark) [inline keyboards/generic_panda/panda65_01/readme.md:15]:
```suggestion
    make generic_panda/panda65_01:default

Flashing example for this keyboard:

    make generic_panda/panda65_01:default:flash
```

File: keyboards/generic_panda/panda65_01/keymaps/default/readme.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+![panda_01 layout image](https://i.imgur.com/fPBUDMT.png)
+
+# The default keymap for Panda_01
```
Comment (noroadsleft) [inline keyboards/generic_panda/panda65_01/keymaps/default/readme.md:3]:
```suggestion
# The default keymap for Panda_01

![panda_01 layout image](https://i.imgur.com/fPBUDMT.png)
```

File: keyboards/generic_panda/panda65_01/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+    ),
+    //fn layer 1
+    [1] = LAYOUT_split_bs(
+        KC_TILD,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,    KC_F12,    KC_F13,     KC_INS,  KC_END,
+        KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,   KC_TRNS,   KC_TRNS,             KC_TRNS,
+        KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,   KC_TRNS,                        KC_TRNS,
+        KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,  KC_TRNS,
+        KC_TRNS,           KC_TRNS,                            KC_TRNS,                                     KC_TRNS,              KC_TRNS,   KC_TRNS,  KC_TRNS
+    ),
+    //fn layer 2
```
Comment (noroadsleft) [inline keyboards/generic_panda/panda65_01/keymaps/default/keymap.c:35]: Regarding Lines 35-50, including these two keymap layers will make the default keymap compile to a larger flash size than is needed, for no benefit or change in functionality.

File: keyboards/generic_panda/panda65_01/readme.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+
+Make example for this keyboard's default keymap (after setting up your build environment):
+
+    make generic_panda/panda65_01:default
+
+Flashing example for this keyboard:
+
+    make generic_panda/panda65_01:default:flash
+
+
```
Comment (fauxpark) [inline keyboards/generic_panda/panda65_01/readme.md:17]:
```suggestion
```

**Comment by noroadsleft** [comment]
https://github.com/qmk/qmk_firmware/pull/10006#discussion_r471376252 is still outstanding.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**